policy/datasets.yaml

```diff
@@ -1038,4 +1038,43 @@ dcrDiff_kernelSrc:
     persistable:   SourceCatalog
     storage:       FitsCatalogStorage
     tables:        'raw'
-
+makeBrighterFatterKernel_config:
+    description: "Configuration for the brighter-fatter coefficient calculation code."
+    persistable: Config
+    python: lsst.cp.pipe.brighterFatterKernelGeneration.MakeBrighterFatterKernelTaskConfig
+    storage: ConfigStorage
+    tables:
+    - raw
+    - raw_visit
+    template: config/makeBrighterFatterKernel.py
+makeBrighterFatterKernel_metadata:
+    description: "Metadata for the brighter-fatter coefficient calculation code."
+    persistable: PropertySet
+    storage: BoostStorage
+    python: lsst.daf.base.PropertySet
+    tables: raw
+    template: brighterFatter_metadata/brighterFatter_metadata.boost
+brighterFatterKernel:
+    description: >
+        New brighter-fatter kernels, such that they do not conflict with brighterFatterKernel in obs_subaru.
+        This will be renamed, and replace the brighterFatterKernel later, but they must exist simultaneously
+        for now.
+    persistable: ignored
+    storage: PickleStorage
+    python: lsst.cp.pipe.makeBrighterFatterKernel.BrighterFatterKernel
+    tables: raw
+    template: ''
+brighterFatterGain:
+    description: "The per-amplifier gains, as calculated and used by the brighter-fatter task."
+    persistable: ignored
+    storage: PickleStorage
+    python: numpy.array
+    tables: raw
+    template: ''
+plotBrighterFatterPtc:
+    description: "Plots of the photon transfer curve, as made by the brighter-fatter task."
+    persistable: None
+    python: builtins.str
```

Comment:
Is `str` the right type for MatplotlibStorage? That seems a bit strange. If you are saying "arbitrary binary data" then wouldn't `builtins.bytes` be better?

Reply:
Just confirming that `None` does indeed work fine (@TallJimbo) , and have changed here.